scripts/recovery/protect-and-retire-noncanonical-worktrees.sh

```
@@ -0,0 +1,392 @@
+    BACKUP_REF["$repo"]="$branch"
+    return
+  fi
+
+  if [[ "$repo" == "$ROOT" ]] && branch_contains_oversized_arbor_db "$repo" "$branch"; then
+    printf 'Skipped unsanitized branch containing .arbor/cache/db: %s\n' "$branch"
+    return
+  fi
+
+  fallback="$(make_recovery_branch "$repo" "$(safe_name "$branch")-push")"
+  git -C "$repo" push -u origin "$fallback" --no-verify
+  BACKUP_REF["$repo"]="$fallback"
```

Shellcheck (0.11.0)

[warning] BACKUP_REF appears unused. Verify use (or export if used externally).

(SC2034)